pkg/domain/domain.go

```diff
@@ -2350,6 +2347,9 @@ func (do *Domain) UpdateTableStatsLoop(ctx, initStatsCtx sessionctx.Context) err
 	variable.EnableStatsOwner = do.enableStatsOwner
 	variable.DisableStatsOwner = do.disableStatsOwner
 	do.statsOwner = do.newOwnerManager(handle.StatsPrompt, handle.StatsOwnerKey)
+	if intest.InTest {
+		do.statsOwner.SetListener(do.ddlNotifier)
```

Comment:
will statsOwner have more logic to be put in Listener? If so, we'd better create a separate struct for Listener and let it invokes DDLNotifier

Reply:
Currently no, but we can always change it later.